src/clndr.js

```diff
@@ -255,6 +256,19 @@
                     this.month
                         .set('month', endMoment.month())
                         .set('year', endMoment.year());
+
+                    // Check the intervalStart is not later than now.
+                    if (this.intervalStart.isAfter(this.month, 'month')) {
+                        this.intervalStart
+                            .set('month', this.month.month())
+                            .set('year', this.month.year());
+                    }
```

Comment:
@dimko 

1. Is there a reason why this check (against intervalStart) is within the `constraints.endDate` block? It seems like it should be in the previous if block.
2. The interval stuff is the most confusing for me, but I don't think I understand why this is included. How does this affect a weekly calendar (instead of a monthly calendar) where the start interval could be after the first day of the month?

Sorry it's been a few months since I've been close to this part, but I just wanted to be sure before updating it.